### DISCUSSION TITLE
PyRx is taken on PyPi

### DISCUSSION THREAD

**Discussion by CEXT-Dan** [CEXT-Dan/PyRx#152] [Ideas]
AcPyRx,  short for autocad python runtime extention
PyCadRx, short for python cad runtime extention 

should I change the name of the repository to match?

**Comment by gswifort** [comment]
Or simply "pycad" - as a comprehensive python library for CAD :)

**Comment by gswifort** [comment]
It is not necessary to change the repository name. There are many projects with different package names.

wxpython → wx
pywin32 →win32com etc.

**Comment by CEXT-Dan** [comment]
I think we have a good case for pyrx, the old pyrx is Python2.7, which isn’t even supported anymore
I will wait for their decision  

Pycad is taken, thinking I will keep pyrx somewhere in the name i.e. pyrx-cad, just so it shows up in search results 
we will see

**Comment by schoeller** [comment]
Whatabout 

PrYx = PYthoning Runtime eXtensions for CAD

Or if too insinuating maybe

RpyX  = Runtime eXtension for CAD PYthoned

Or rxcad or cadrx

**Comment by CEXT-Dan** [comment]
added cad-pyrx
https://pypi.org/project/cad-pyrx/
